just-install.json

```diff
@@ -1266,6 +1266,19 @@
       },
       "version": "1.7.13"
     },
+    "mpv": {
+      "version": "latest",
+      "installer": {
+        "x86": "https://downloads.sourceforge.net/project/mpv-player-windows/bootstrapper.zip",
+        "kind": "custom",
+        "options": {
+          "arguments": [
+            "powershell",
+            "Add-Type -as System.IO.Compression.FileSystem; [System.IO.Compression.ZipFile]::ExtractToDirectory('{{.installer}}', '{{.PROGRAMFILES}}\\mpv'); cd '{{.PROGRAMFILES}}\\mpv'; cmd /c start cmd /k updater.bat"
```

Comment:
System.IO.Compression.ZipFile requires .net framework 4.5.

Reply:
...Damn. Any suggestions on alternatives? I'm really not sure what else to even try here. There is a way to download the mpv archives directly -- without even touching the updater -- but they are all in 7z format. My guess is that relying on 7z being installed is also a bad idea? I could essentially convert the arguments into something along the lines of `7z x -o {{.PROGRAMFILES}}\\mpv {{.installer}}` quite easily, but of course, 7z would need to exist first and foremost.

Reply:
IIRC (I'm mainly using linux), you can do it in VBScript with Scripting.FileSystemObject, but it's probably not worth the effort. @lvillani what do you think?

Reply:
The vbscript option would mean messing around with a custom installer script because vbscript has no way to feed it the script via arguments. At that point, I'd much rather bundle a .bat script together with wget/curl and 7zip to an .exe with something like [this](http://f2ko.de/en/b2e.php) to handle the download and extraction, or even just write a custom installer in a sane programming language that would compile to an .exe. Would that be acceptable? It would no longer be an official MPV package, I'd host it on my GitHub or something, but it would still download and extract all the same things.

Reply:
I would only accept official packages but even mpv's `bootstrapper.zip` feels sketchy enough that it probably crosses the line of being unacceptable.

I can't fathom why they don't create an installer package with InnoSetup/NSIS/WiX like almost everyone else does instead of doing this 7zip/bootstrapper.zip nonsense.

Reply:
That's fair enough. I'm just going to close this, then, as there doesn't seem to be a sensible way to "just install" this particular program.